lingua_franca/parse.py

```diff
@@ -200,9 +201,11 @@ def extract_duration(text, lang=None):
         return extract_duration_en(text)
     if lang_code == "cs":
         return extract_duration_cs(text)
+    if lang_code == "de":
+        return extract_duration_cs(text)
```

Comment:
Howdy, and thanks for this! Couple of things that, frankly, remind me of myself 😛 

`return extract_duration_de(text)`